frontend/src/App.js

```diff
@@ -23,8 +23,21 @@ import SideNav, { NavItem, NavIcon } from '@trendmicro/react-sidenav';
 import '@trendmicro/react-sidenav/dist/react-sidenav.css';
 import Center from 'react-center';
 
+import Card from './routes/Components/Card'
+
+const logo = {
+  width: '3.125rem', 
+  paddingTop: '0.625rem', 
+  paddingBottom: '0.3125rem'
+}
+
+const paddingBottom={
+  paddingBottom: '50vh' 
+}
+
+
 const LoginContainer = () => (
-  <div className="App-header" style={{ backgroundColor: 'white' }}>
+  <div className="App-header loginBackground">
```

Comment:
is this a style that's shared btwn client and advisor and thats why its in styles.css instead of a const in this file?

Reply:
**Just changed this to use const**